addon/components/form-paper.js

```diff
@@ -0,0 +1,28 @@
+import Component        from 'ember-component';
+import textInputMixin   from 'ember-railio-form-components/mixins/text-input-mixin';
+import formFieldOptions from
+  'ember-railio-form-components/mixins/form-field-options';
+
+import layout from '../templates/components/form-paper';
+
+
+export default Component.extend(textInputMixin, formFieldOptions, {
+  classNames: ['text-field'],
```

Comment:
When this is a text-field, form-paper is not a right name. I prefer paper-form-field anyway